systemtests/src/test/java/io/enmasse/systemtest/common/BrokeredPersistentMessagesTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.common;
```

Comment:
Likewise, shouldn't this be io.enmasse.systemtest.brokered

Reply:
I leaved it under `common` package to keep the tests in the same package as it was previously when inside CommonTest

Reply:
Don't we have separate CI profilesfor brokered and standard? Won't this mean unnecessarily we run the brokered test when running the standard suite and vice versa?